src/vm/codeman.cpp

```diff
@@ -1016,6 +1016,35 @@ PTR_VOID GetUnwindDataBlob(TADDR moduleBase, PTR_RUNTIME_FUNCTION pRuntimeFuncti
 
     *pSize = size;
     return xdata;
+
+#elif defined(_TARGET_ARM64_)
+    // compute the size of the unwind info
+    PTR_TADDR xdata = dac_cast<PTR_TADDR>(pRuntimeFunction->UnwindData + moduleBase);
+    ULONG epilogScopes = 0;
+    ULONG unwindWords  = 0;
+    ULONG size = 0;
+
+    if ((xdata[0] >> 27) != 0) {
```

Comment:
This should be xdata[0] >> 22. This needs to be redone completely with better understanding. copy paste from arm will not work here.